src/main/java/uk/gov/hmcts/reform/civil/constants/SdoR2UiConstantSmallClaim.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.civil.constants;
+
+public final class SdoR2UiConstantSmallClaim {
+
+    private SdoR2UiConstantSmallClaim() {
+        //To satisfy sonarQube
+    }
+
+    public static final String JUDGE_RECITAL = "Upon considering the statements of case and the information provided by the parties,";
+    public static final String PPI_DESCRIPTION = """
+        send to the Claimant(s):
+        a) A schedule of the following amounts itemised by date;
+        i) PPI premium charged,
+        ii) Commission receivable by the Defendant(s),
+        iii) Rate of commission as a percentage of the premium charged,
+        iv) Totals and average commission rates, in respect of the whole period to which the claim relates,
+        v) Contractual interest associated with each such premium charged.
+        b) A copy of the credit agreement;
+        c) A copy of the PPI policy application and agreement;
+        d) Any cancellation notices;
+        e) Any correspondence in connection with a complaint by the Claimant(s);
+        f) A copy of any correspondence as to redress if not included in (e);
+        g) The amount, if any, of redress (under the FCA scheme or otherwise) and the date it was paid.
+
+        The parties shall endeavour to agree a calculation of the sums in issue and upload to the Digital Portal the agreed calculation.\
+         In default of agreement each party shall upload to the Digital Portal its own calculation not later than 7 days before any hearing.\
+         The calculation must show the amount which is intended to remove any unfairness from the relationship,broken down into:
```

Comment:
A space after the comma is missing. 
"relationship,broken"

Reply:
> A space after the comma is missing. "relationship,broken"

Done